Nos/Views/Components/RepliesLabel.swift

```diff
@@ -17,7 +17,7 @@ struct RepliesLabel: View {
         self.repliesDisplayType = repliesDisplayType
 
         let noteIdentifier = note.identifier ?? ""
-        _replies = Event.replies(to: noteIdentifier)
+        _replies = FetchRequest(fetchRequest: Event.replies(to: noteIdentifier))
```

Comment:
This was what happened when I saw Event+CoreDataClass importing SwiftUI and removed it.